test-setup.js

```diff
@@ -4,7 +4,7 @@ import { configure } from 'enzyme';
 import AdapterReact15 from 'enzyme-adapter-react-15';
 import AdapterReact16 from 'enzyme-adapter-react-16';
 
-if (process.env.REACT_VERSION === 15) {
+if (process.env.REACT_VERSION.startsWith('^15.')) {
```

Comment:
Is it a choice to have the wildcard (so no fixed version) ? 
Just a question, it can be a choice here.